src/Helper/EditorHelper.php

```diff
@@ -46,18 +46,13 @@ public function fromString($string)
             throw new \RuntimeException('No EDITOR environment variable set.');
         }
 
-        if (defined('PHP_WINDOWS_VERSION_MAJOR')) {
```

Comment:
This can give problems when used in Docker. I will change this to detect if Docker is used, and use tty then.
I rather don't want to remove this as I'm using Gush without Docker and have an EDITOR configured.